### PR TITLE
CORTX_30942

### PR DESCRIPTION
**Pull request by ankita-93**
Signed-off-by: ankita-93 <ankita.thummar@seagate.com>

# Problem Statement
- Updates in test as per change in delete_kpod_with_shutdown_methods() under [https://jts.seagate.com/browse/CORTX-30842](url)

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: tests/ha/test_server_pod_failure.py [hunk 1]
```diff
@@ -2179,52 +2021,31 @@ def test_ios_unsafe_server_pod_shutdown(self):
...
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 5: Services on remaining pods are in online state")
+        self.restore_method = resp[1][pod_name]['method']
+        assert_utils.assert_true(resp[0], "Cluster/Services status is not as expected")
+        LOGGER.info("Step 2: Successfully shutdown server pod %s safely. Verified cluster "
+                    "has some failure & remaining pods status is online.", pod_name)
 
-        LOGGER.info("Step 6: Perform WRITEs-READs-Verify-DELETEs with variable object sizes. 0B + ("
+        LOGGER.info("Step 3: Perform WRITEs-READs-Verify-DELETEs with variable object sizes. 0B + ("
```
Comment (ApurwaMohite) [inline tests/ha/test_server_pod_failure.py:2040]: Please remove workload sizes from logger. 
Check at all the places.

File: tests/ha/test_server_pod_failure.py [hunk 2]
```diff
@@ -2696,53 +2474,30 @@ def test_server_pod_failure(self):
...
-        server_pod_list.remove(server_pod_name)
-        LOGGER.info("Step 5: Check services status on remaining pods %s", server_pod_list)
-        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=server_pod_list, fail=False)
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 5: Services of remaining pods are in online state")
+        self.restore_method = resp[1][pod_name]['method']
+        assert_utils.assert_true(resp[0], "Cluster/Services status is not as expected")
+        LOGGER.info("Step 2: Successfully shutdown server pod %s safely. Verified cluster "
+                    "has some failure & remaining pods status is online.", pod_name)
```
Comment (ApurwaMohite) [inline tests/ha/test_server_pod_failure.py:2490]: Instead of "Verified cluster has some failure" please add cluster and services states are as expected.

**Comment by ankita-93** [comment]
Collection logs snap
image

File: tests/ha/test_server_pod_failure.py [hunk 3]
```diff
@@ -1408,8 +1326,8 @@ def test_reads_writes_during_server_pod_down(self):
         LOGGER.info("STARTED: Test to verify READs/WRITEs during server pod down by "
                     "delete deployment.")
         event = threading.Event()  # Event to be used to send intimation of server pod deletion
-        LOGGER.info("Step 1: Perform continuous READs/WRITEs with variable object sizes. 0B + ("
-                    "1KB - 512MB) during server pod down by delete deployment.")
+        LOGGER.info("Step 1: Perform continuous READs/WRITEs with variable object sizes during "
```
Comment (sonalk0209) [inline tests/ha/test_server_pod_failure.py:1329]: either keep continuous or during

**Review by sonalk0209 — APPROVED** [review]
@ankita-93 minor comment and add collections log

**Review by ApurwaMohite — APPROVED** [review]

**Pull request merged**